app/pages/Accounts/AccountPage.tsx

```diff
@@ -1,23 +1,41 @@
-import React from 'react';
-import { Switch, Route } from 'react-router-dom';
+import React, { useEffect, useState } from 'react';
 import { useDispatch, useSelector } from 'react-redux';
-import PlusIcon from '@resources/svg/plus.svg';
+import { Dispatch } from 'redux';
 import { push } from 'connected-react-router';
-import AccountList from './AccountList';
-import AccountView from './AccountView';
+
 import NoIdentities from '~/components/NoIdentities';
-import { accountsSelector } from '~/features/AccountSlice';
-import routes from '~/constants/routes.json';
+import {
+    accountsSelector,
+    loadAccountInfos,
+    loadAccounts,
+} from '~/features/AccountSlice';
 import MasterDetailPageLayout from '~/components/MasterDetailPageLayout';
-import BuildSchedule from './BuildSchedule';
-import AccountPageHeader from './AccountPageHeader';
-import PageLayout from '~/components/PageLayout';
+import { RootState } from '~/store/store';
+import SimpleErrorModal from '~/components/SimpleErrorModal';
+import routes from '~/constants/routes.json';
 
-const { Header, Master, Detail } = MasterDetailPageLayout;
+import useAccountSync from './useAccountSync';
+import AccountListPage from './AccountListPage';
+import AccountDetailsPage from './AccountDetailsPage';
+
+const { Header } = MasterDetailPageLayout;
+
+async function load(dispatch: Dispatch) {
+    const accounts = await loadAccounts(dispatch);
+    return loadAccountInfos(accounts, dispatch);
+}
 
 export default function AccountsPage() {
-    const dispatch = useDispatch();
     const accounts = useSelector(accountsSelector);
+    const dispatch = useDispatch();
+    useAccountSync();
+    const { simpleView } = useSelector((s: RootState) => s.accounts);
```

Comment:
hmm

Reply:
hmm?

Reply:
I'm confused why you are using a selector on the entire accounts state, and _then_ destructing it, instead of just selecting the boolean only in the selector.

Reply:
Because it's shorter 😄 what difference does it make?

Reply:
hmm... good point, hadn't thought about that